.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+<!--- PR title format: [GH#<gh-issue-id>][BZ#<bz-issue-id>][<jira-issue-id>]: <short-description-of-the-pr> --->
+
+Version:
+<!--- What version or versions of OpenShift does this apply to? --->
+
+Issue:
+<!--- Add a link to the Bugzilla, Jira, or GitHub issue. --->
+
+Link to docs preview:
+<!--- Add direct links to the Netlify preview build. --->
+
+Description:
+
```

Comment:
- This field is the only one that doesn't have an actionable sentence below it. 
- For this one, honestly I don't expect people to add a description when it is often in the commit/PR title. Maybe we could call it "Additional information" or something like that instead, and they can add a description or other info they want to pass along under it?